docs/ref/django-admin.txt

```diff
@@ -1650,6 +1650,16 @@ Example usage::
 
     django-admin runserver --no-color
 
+.. django-admin-option:: --force-color
+
+Force colorized command output.
+
+Example usage::
+
+    django-admin runserver --force-color
+
+.. versionadded:: 2.2
```

Comment:
I think `versionadded` clause should be added just below `.. django-admin-option :: --force-color`.

Reply:
It seems a blank line before `versionadded` clause need is necessary.

Reply:
Yes, but it should be moved just after `.. django-admin-option :: --force-color` i.e.
```
.. django-admin-option:: --force-color
 
.. versionadded:: 2.2
```

Currently it is at the end of an option description which is not correct, it looks like


![obraz](https://user-images.githubusercontent.com/2865885/45241216-d119fc80-b2eb-11e8-8217-d9b0c272b41a.png)

instead of

![obraz](https://user-images.githubusercontent.com/2865885/45241625-6cf83800-b2ed-11e8-8ecb-5b1f7fac737e.png)